src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX = 100;
+    ColorSupplier colorSupplier = new ColorSupplier();
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(),
+                    new Random().nextInt(MAX));
```

Comment:
Introduce `Random` as field variable, you don't need to create new object every time you call `getRandomBall`